lasagne/layers/recurrent.py

```diff
@@ -564,9 +550,9 @@ def get_output_for(self, input, mask=None, *args, **kwargs):
         # Because the input is given for all time steps, we can precompute
         # the inputs to the gates before scanning.
         # input is dimshuffled to (n_time_steps, n_batch, n_features)
-        # W_in_to_gates is (n_features, 4*num_units). input_dot_W is then
+        # W_in_to_gates is (n_features, 4*num_units). input_dot_w is then
```

Comment:
I can do it, I just wasn't sure what version you prefered :) 

Just so we are clear:

`input_dot_W` and `input_dot_W_n` inside scan?

I think we should squash the commits at the end anyway?